src/main/java/com/hm/achievement/gui/CategoryGUI.java

```diff
@@ -196,20 +191,20 @@ private void displayPage(String categoryName, Player player, Map<String, Long> s
 			String achName = plugin.getPluginConfig().getString(categoryName + '.' + path + ".Name", "");
 			String receptionDate = plugin.getDatabaseManager().getPlayerAchievementDate(player.getUniqueId(), achName);
 
-			boolean inelligibleSeriesItem = true;
+			boolean ineligibleSeriesItem = true;
```

Comment:
This was added by #62, I plead not guilty for this spelling mistake. :stuck_out_tongue: